src/store/amazon.ts

```diff
@@ -0,0 +1,56 @@
+import {Store} from './store';
+
+export const amazon: Store = {
+	cartUrl: '',
+	links: [
+		{
+			brand: 'PNY',
```

Comment:
Do you mind lowercasing everything here?

Reply:
nope